### DISCUSSION TITLE
How do I chose witch field used in realations

### DISCUSSION THREAD

**Discussion by H1Mohamed** [pocketbase/pocketbase#3652] [Q&A]
I have a relation and I want to select witch field is used to extract data from,how can I do that???
I'm new to pocketbase and db's overall, so my understanding of realtions in pocketbase might be wrong so please correct me if I am

**Comment by ganigeorgiev** [comment]
All relations are associated via id. This can't be changed at the moment.

If you just want to specify which field value(s) to show in the Admin UI table listing, you can go to your related collection and mark the fields you want to be used as "Presentable".

![presentable_field_option](https://i.imgur.com/5rhM8CU.png)

**Reply by H1Mohamed** [reply]
what does that do?

**Reply by ganigeorgiev** [reply]
In the Admin UI listings where the relation is visualized it will show the selected "Presentable" field value(s). If no explicit "Presentable" field(s) is selected, we try to "detect" commonly used presentable fields like "title", "name", "label", etc. or just fallback to the id.

If that's not what you are looking for, please elaborate and/or provide an example with the desired outcome.

**Reply by H1Mohamed** [reply]
well I have a posts collection, I need the username field to be a relation to the username field in the users' collection, how can I do that?
is there is any way to do it???

**Comment by H1Mohamed** [comment]
well after some thinking, the relation is to link a record to a record from an other collection, so the real question is how do I make a copy of a record's fields' data in another record and inshore that it's the same
